Sprint-1/implement/dedupe.js

```diff
@@ -1 +1,12 @@
-function dedupe() {}
+function dedupe(a) {
+  if (a === undefined || a.length == 0) {
+    return "given an empty array, it returns an empty array";
```

Comment:
I think the "empty array" in the spec means an array with 0 element, as in `[]`.